CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   of the commit log.
 
 ## Unreleased
+
+* Add cookie category override function [PR #1312](https://github.com/alphagov/govuk_publishing_components/pull/1312)
```

Comment:
Just noticed that the formatting of these entries changed midway through 21.24.0, not sure if it was intentional. Used to be `PR name ([PR num](link))`